app/controllers/concerns/fully_authenticatable.rb

```diff
@@ -3,6 +3,10 @@
 module FullyAuthenticatable
   def delete_branded_experience(logout: false)
     ServiceProviderRequestProxy.delete(request_id)
+    if session[:sp]
+      session[:sp][:successful_handoff] = true
+    end
```

Comment:
I don't know that I'd see this always as a necessary side-effect of "deleting the branded experience". For example, we [delete the branded experience when a user signs out](https://github.com/18F/identity-idp/blob/ad5524df61a945c056db2c4b983e96dc7e59052e/app/controllers/sign_out_controller.rb#L12), but that's not a successful handoff. I might suggest moving this up to where we call `deleted_branded_experience` as part of the [OIDC](https://github.com/18F/identity-idp/blob/ad5524df61a945c056db2c4b983e96dc7e59052e/app/controllers/openid_connect/authorization_controller.rb#L129)/SAML handoff logic.